jwql/utils/utils.py

```diff
@@ -561,8 +561,8 @@ def get_rootnames_for_instrument_proposal(instrument, proposal):
     rootnames : list
         List of rootnames for the given instrument and proposal number
     """
-    tap_service = vo.dal.TAPService("http://vao.stsci.edu/caomtap/tapservice.aspx")
-    tap_results = tap_service.search(f"select observationID from dbo.CaomObservation where collection='JWST' and maxLevel=2 and insName like '{instrument.lower()}' and prpID='{int(proposal)}'")
+    tap_service = vo.dal.TAPService("https://vao.stsci.edu/caomtap/tapservice.aspx")
```

Comment:
Security is a good thing.

Reply:
When I was running the code manually, the ST firewall was not allowing the use of http. It kept crashing the script. I'm not sure why it was crashing for me yesterday, but hasn't been crashing on the JWQL servers. Either way, https seems to be the answer.